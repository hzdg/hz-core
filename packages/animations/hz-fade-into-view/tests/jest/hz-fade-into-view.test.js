/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import FadeIntoView from '../../src';

describe('FadeIntoView', () => {
  it('should render something', () => {
    const item = (
      <FadeIntoView direction="Up" activate>
        <div>Boo!</div>
      </FadeIntoView>
    );
    const renderedItem = renderer.create(item).toJSON();
    expect(renderedItem).toMatchSnapshot();
  });
});
