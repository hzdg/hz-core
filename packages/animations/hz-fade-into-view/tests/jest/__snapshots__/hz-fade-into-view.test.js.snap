// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FadeIntoView should render something 1`] = `
<div
  style={
    Object {
      "opacity": 0,
      "transform": "translateY(100px)",
    }
  }
>
  <div>
    Boo!
  </div>
</div>
`;
